Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -225,19 +239,42 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath, b
     %% 6.   Initialize quantification parameters
     if ~isfield(x.Q,'nCompartments') || isempty(x.Q.nCompartments) || x.Q.nCompartments>2
         x.Q.nCompartments = 1; % by default, we use a single-compartment model, as proposed by the Alsop et al. MRM 2014 concensus paper
-    end
+	end
 
-    if ~isfield(x.Q,'ATT')
+	if ~isfield(x.Q,'ATT')
         x.Q.ATT = 1800; % ms as default micro-vascular ATT
-    end
-    if ~isfield(x.Q,'TissueT1')
-        x.Q.TissueT1 = 1240; % T1 GM tissue @ 3T
-    end
-    if ~isfield(x.Q,'BloodT1')
-        x.Q.BloodT1  = 1650; % T1 relaxation time of arterial blood DEFAULT=1650 @ 3T
-    end
+	end
+	if ~isfield(x.Q,'TissueT1')
+		switch(x.MagneticFieldStrength)
+			case 3
+				x.Q.TissueT1 = 1240; % T1 GM tissue @ 3T
+			case 1
+				x.Q.TissueT1 = 1000; 
+			case 4
+				x.Q.TissueT1 = 1350;
+			otherwise
+				x.Q.TissueT1 = 1240;
+				fprintf('%s\n',['Warning: Unknown T1 GM for ' num2str(x.MagneticFieldStrength) 'T scanners']);
```

Comment:
Warning is perhaps not needed when not using TissueT1. `x.Q.TissueT1` is used for two-compartment model or for M0 correction.

Reply:
We can keep warning since Wibeke gave me a reference for almost all B0s - check the new values + references